docs/03-pages/02-api-reference/03-next-config-js/serverExternalPackages.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: serverExternalPackages
+description: Opt-out specific dependencies from the dependency bundling enabled by `bundlePagesRouterDependencies`.
+---
+
+Opt-out specific dependencies from being included in the automatic bundling of the [`bundlePagesRouterDependencies`](/docs/pages/api-reference/next-config-js/bundlePagesRouterDependencies) option.
+
+These pages will then use native Node.js `require` to resolve the dependency.
+
+```js filename="next.config.js"
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  serverExternalPackages: ['@acme/ui'],
+}
+
+module.exports = nextConfig
+```
+
+Next.js includes a [short list of popular packages](https://github.com/vercel/next.js/blob/canary/packages/next/src/lib/server-external-packages.json) that currently are working on compatibility and automatically opt-ed out:
+
```

Comment:
since it's an mdx file so u can import the json file where these deps names are written and map to display it here. This way both the lists will stay synced

Reply:
I'd consider that out of scope for this PR. We have the validate script that manages it for now. It'd be a good follow up change!